pom.xml

```diff
@@ -152,6 +152,10 @@
            <target>1.8</target>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M6</version>
```

Comment:
I think M7 is the latest now

Reply:
Thank you both. I switched to M7. When I ran `mvn clean test` it didn't run any tests; so I simplified JUnit deps until it did.